gitops/openshift/apps/scss-jag/template/scss_jag.env

```diff
@@ -0,0 +1,7 @@
+APP_NAME=jag-scss
+COMPONENT_NAME=jag-scss
+OC_NAMESPACE=17b24e
```

Comment:
I've created a secret for this: OPENSHIFT_SA_NAMESPACE - should we use this instead?

Reply:
The templates are checked-in in source code just for source controlling. Actually we use these templates to deploy manually in Openshift using OC command. The .env files with values will be fed to the template for deployment manually in openshift.

So, values from github secrets can't be used and those secrets are used only for github actions.